actions.php

```diff
@@ -34,7 +34,7 @@
     $options['fields'][$field_name] = 1;
   }
 
-  $app->trigger("moderation.find.before", [$name, &$options]);
+  $app->trigger("moderation.collections.find.before", [$name, &$options]);
```

Comment:
this is a breaking change, would be possible to maintain the `moderation.find.before` hook and then call the functionality depending on what we have there? its fine on having the new hooks, just concerned on removing the existing one as any addons that rely on it will break